#! /usr/bin/env python


# Initialize ros and import parameter generator
PACKAGE='mouse_perturbation_robot'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


# Add parameters
#       Name     Type    Level Description     	Default 	Min   	Max
gen.add("alpha",double_t,0,"Low pass filter gain",0.9,0.8,1)
gen.add("useMovingAverage",bool_t,0,"Use moving average",False)
gen.add("windowSize",int_t,0,"Size of moving average",6,4,10)


# Generate files
exit(gen.generate(PACKAGE, "mouse_perturbation_robot", "mouseInterface_params"))